packages/editor-ui/src/views/SettingsLogStreamingView.vue

```diff
@@ -173,6 +171,13 @@ export default mixins().extend({
 		},
 	},
 	methods: {
+		onDestinationWasSaved() {
+			this.$forceUpdate()
```

Comment:
linting complaining about the missing semicolon here

Reply:
Damnit, my VSCode eslint keeps failing me 😠 Should be fixed now